src/SDDPoptimize.jl

```diff
@@ -27,28 +31,29 @@ fulfilled.
     `n` iterations, where `n` is the number specified by display.
 
 # Returns
-* `V::Array{PolyhedralFunction}`:
-    the collection of approximation of the bellman functions
-* `problems::Array{JuMP.Model}`:
-    the collection of linear problems used to approximate
-    each value function
-* `sddp_stats::SDDPStat`: statistics of the algorithm run
+`SDDPInterface`
 
 """
-function solve_SDDP(model::SPModel, param::SDDPparameters, verbose=0::Int64)
-    check_SDDPparameters(model,param,verbose)
-    # initialize value functions:
-    V, problems = initialize_value_functions(model, param)
-    (verbose > 0) && println("Initial value function loaded into memory.")
+function solve_SDDP(model::SPModel, param::SDDPparameters, verbose=0::Int64;
+                    stopcrit::AbstractStoppingCriterion=IterLimit(20),
+                    prunalgo::AbstractCutPruningAlgo=CutPruners.AvgCutPruningAlgo(-1),
```

Comment:
If I am a new user I might find disturbing to find that I need to install a new package (CutPruners) even if I don't want to use it. 

I have to find out if there is a way to check if a package is installed and load and use it in this case without having it in REQUIRE. Same thing apply to plot packages.